testgrid/config/config.yaml

```diff
@@ -865,7 +865,9 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-latest-upgrade-cluster
 - name: kubernetes-e2e-gke-latest-upgrade-master
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gke-latest-upgrade-master
-
+# charts tests
+- name: ci-kubernetes-e2e-charts-gce
+  gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-charts-gce
```

Comment:
Thanks for the quick review :) Anything else I've missed? 
I don't think I can push a new image of e2e-runner. Is that something someone else can help me do after I've addressed all comments?